src/cares_wrap.cc

```diff
@@ -270,31 +285,43 @@ void ares_sockstate_cb(void* data,
 }
 
 
-Local<Array> HostentToAddresses(Environment* env, struct hostent* host) {
+Local<Array> HostentToAddresses(Environment* env,
+                                struct hostent* host,
+                                Local<Array> prev = Local<Array>()) {
   EscapableHandleScope scope(env->isolate());
-  Local<Array> addresses = Array::New(env->isolate());
+  auto context = env->context();
+  Context::Scope context_scope(context);
```

Comment:
I am pretty sure you don’t need these if you are already in *some* other `Context::Scope` :)